docs/docs/developers/contracts/writing_contracts/historical_data/slow_updates_tree/implement_slow_updates.md

```diff
@@ -0,0 +1,63 @@
+---
+title: How to implement a Slow Updates Tree
+---
+
+To learn more about the Slow Updates Tree, go [here](./main.md)
+
+1. Copy the *SlowTree.nr* example and its dependencies, found [here](https://github.com/AztecProtocol/aztec-packages/tree/master/yarn-project/noir-contracts/contracts/slow_tree_contract). Replace the constants with whatever you like and deploy it to your sandbox
+2. Copy the *SlowMap interface* for easy interaction with your deployed SlowTree. Find it [here](https://github.com/AztecProtocol/aztec-packages/blob/master/yarn-project/noir-contracts/contracts/token_blacklist_contract/src/interfaces.nr)
+3. Import this interface into your contract
+
+#include_code interface yarn-project/noir-contracts/contracts/token_blacklist_contract/src/main.nr rust
+
+5. Create a storage init function for the same value in both public and private storage
+
+#include_code slow_updates_storage yarn-project/noir-contracts/contracts/token_blacklist_contract/src/main.nr rust
```

Comment:
`slow_updates_storage` doesn't exist, but perhaps is referring to https://github.com/AztecProtocol/aztec-packages/blob/docs/dip1-howtos/yarn-project/noir-contracts/contracts/token_blacklist_contract/src/main.nr#L84-L101

Reply:
ah it used to exist but was abstracted a few days ago. removed